react-backend/client/src/components/admin/adminUserDetailsPage.js

```diff
@@ -83,28 +83,31 @@ export default class AdminUserDetailsPage extends Component {
   handleSubmit = (event) => {
     event.preventDefault();
 
-    adminPost(this, '/api/user/details/update', {
-      userId: this.state.userId,
-      userFullName: this.state.userFullName,
-      sex: this.state.sex,
-      yearOfBirth: this.state.yearOfBirth,
+    Visitors.update(this.props.auth, {
+      id: this.state.userId,
+      name: this.state.userFullName,
+      gender: this.state.sex,
```

Comment:
Lol

Reply:
I know, I would have changed the state field too, but wanted to keep the diff small.